src/main/kotlin/com/cognifide/gradle/aem/tooling/tasks/Tail.kt

```diff
@@ -0,0 +1,96 @@
+package com.cognifide.gradle.aem.tooling.tasks
+
+import com.cognifide.gradle.aem.common.AemDefaultTask
+import com.cognifide.gradle.aem.common.AemTask
+import com.cognifide.gradle.aem.instance.Instance
+import com.cognifide.gradle.aem.tooling.tail.Log
+import com.cognifide.gradle.aem.tooling.tail.LogDestination
+import com.cognifide.gradle.aem.tooling.tail.LogSource
+import com.cognifide.gradle.aem.tooling.tail.Tailer
+import java.io.BufferedReader
+import java.io.File
+import java.io.FileWriter
+import java.io.InputStreamReader
+import java.net.HttpURLConnection
+import java.net.URL
+import java.util.*
+import kotlinx.coroutines.*
+import org.gradle.api.tasks.TaskAction
+import org.gradle.process.internal.shutdown.ShutdownHooks
+
+open class Tail : AemDefaultTask() {
+
+    class AemLogSource(private val aemInstance: Instance) : LogSource {
```

Comment:
make tasks as lightweight as it is possible. tasks are generally not easy reusable. so that move that inner classes outside

Reply:
OK